.gitignore

```diff
@@ -0,0 +1,6 @@
+obj/
+bin/
+DataFiles/
+.vscode/
+__blobstorage__/
+__queuestorage__/
```

Comment:
This is a bit of a silly request but if you don't have a new line at the end of your file git will create this big red mark haha.  Could you add a new line in any file you see this occurring? :)